app/src/ui/create-branch/create-branch-dialog.tsx

```diff
@@ -395,12 +404,13 @@ export class CreateBranch extends React.Component<
     // fork will have the same default branch name
     if (currentBranchName === upstreamDefaultBranch.nameWithoutRemote) {
       return (
-        <p>
+        <div>
           Your new branch will be based on{' '}
           <strong>{upstreamRepositoryFullName}</strong>
           's {defaultBranchLink} (
           <Ref>{upstreamDefaultBranch.nameWithoutRemote}</Ref>).
-        </p>
+          {this.renderForkLink()}
```

Comment:
Same as comment on `renderRegulartBranchSelection` - Use prefix instead of additional sentence.. probably a good place for another method like `renderForkLinkPrefix`.. just got to be cautious of the 'y' vs 'Y' depending on if it is a fork.